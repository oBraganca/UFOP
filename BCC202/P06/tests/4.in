1 80
1 68
1 65
1 66
1 64
2 33
4 52
2 68
1 24
1 89
1 66
3
4 69
2 22
5
2 87
2 100
6
5
-1