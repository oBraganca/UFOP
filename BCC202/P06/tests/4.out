Nao Pertence
Nao Pertence
24 64 65 66 80 89 
Posicao inicial 1
24 64 65 66 80 89 
